app/src/main/java/com/team7/cmput301/android/theirisproject/model/Record.java

```diff
@@ -7,19 +7,38 @@
 import java.util.ArrayList;
 import java.util.Date;
 
+import io.searchbox.annotations.JestId;
+
+/**
+ * A single record of a particular Problem
+ *
+ * @author Jmmxp
+ */
 public class Record {
+
+    @JestId
+    private String _id;
+    private String problemId;
+
     private String text;
     private String title;
     private Date date;
     private GeoLocation geoLocation;
     private BodyLocation bodyLocation;
     private ArrayList<RecordPhoto> recordPhotos = new ArrayList<RecordPhoto>();
 
-    public Record(String title, String text, Date date, GeoLocation geoPt, BodyLocation bodyPt, ArrayList<RecordPhoto> recordPhotos) {
+    public Record(String problemId, String title, String text, Date date, GeoLocation geoPt, BodyLocation bodyPt, ArrayList<RecordPhoto> recordPhotos) {
+        this.problemId = problemId;
+        this.title = title;
+        this.text = text;
+        this.date = date;
+        this.geoLocation = geoPt;
+        this.bodyLocation = bodyPt;
+        this.recordPhotos = recordPhotos;
```

Comment:
I think that recordPhotos should be separate from the record model as it may be separate in our database, we can add an ArrayList<RecordPhoto> in our controller which can be queried after we query our record. If we are to edit/add a record the POST will include data from recordPhotos attribute which we may not want